packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
`aggregateId` doesn't help when skimming code to know if there's a command or event being generated, nor does it assist with really grasping that commands are always directed at an entity (targets) whereas events are just an announcement by an entity that in doesn't care who is listening. It actually helped me early on to have this formality